sportngin/opsicle
# Fix MFA deploy tracking bug

Description and Impact
----------------------
Fixes https://github.com/sportngin/opsicle/issues/82 by changing the config and client to re-use the session for the whole command.

Deploy Plan
-----------
> Does Platform Operations need to know anything special about this deploy? Are migrations present?

Rollback Plan
-------------
* To roll back this change, revert the merge with: `git revert -m 1 MERGE_SHA` and perform another deploy.

URLs
----
https://github.com/sportngin/opsicle/issues/82

QA Plan
-------
Put in a .opsicle file in the project for testing.
Use an environment with MFA configured.
Try deploy to verify bug is fixed.
`bundle exec opsicle --debug --verbose deploy staging`
Try other commands for regression.
